docs/guides/add-exposure-tests.mdx

```diff
@@ -0,0 +1,94 @@
+---
+title: "Add exposure validation tests"
+---
+
+After you [install the dbt package](/quickstart#install-the-dbt-package), you can add Elementary exposure validation tests.
+
+## Exposure validation dbt tests
+
+Elementary dbt package includes **exposure validation tests, implemented as [dbt tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests)**.
+These tests can detect missing columns and column data types for defined exposures.
+
+## Configure your elementary exposure validation tests
+
+<Tip>
+  You can read up about exposures and how to add them at [the dbt docs
+  site](https://docs.getdbt.com/docs/build/exposures)
+</Tip>
+
+Within your `models` directory, add a file called `exposures.yml`
+
+```yml
+exposures:
+  - name: customers
+    label: Customer Dashboard
+    type: dashboard
+    maturity: high
+    url: https://your.bi.tool/dashboards/1
+    description: >
+      Shows customer growth
+
+    depends_on:
+      - ref('customers')
+
+    owner:
+      name: Callum McData
+      email: data@jaffleshop.com
+    meta:
+      columns:
+        - name: "customer_id"
+```
+
+<Note>You can optionally also specify the column data type (`data_type`)</Note>
+
+```yml
+- name: returned_orders
+  label: Returned Orders
+  type: dashboard
+  maturity: high
+  url: https://your.bi.tool/dashboards/2
+  description: >
+    Returned orders over time
+
+  depends_on:
+    - ref('returned_orders')
+
+  owner:
+    name: Callum McData
+    email: data@jaffleshop.com
+  meta:
+    columns:
+      - name: "order_id"
+        data_type: "numeric"
+```
+
+Then in your module schema, add the elementary exposure tests:
```

Comment:
I think it's important to point out that the test needs to be set up for every model used in the exposure.
Also you might wanna explain a little on how the test works, mainly to communicate why no other setup is needed except this test :)